Carmunication/Controllers/SessionStartViewController.swift

```diff
@@ -7,10 +7,138 @@
+        collectionView.translatesAutoresizingMaskIntoConstraints = false
+        collectionView.register(GroupCollectionViewCell.self, forCellWithReuseIdentifier: "groupCell")
+        collectionView.showsHorizontalScrollIndicator = false   // 스크롤바 숨기기
+        return collectionView
+    }()
     override func viewDidLoad() {
         super.viewDidLoad()
         view.backgroundColor = .systemBackground
+        setCollectionView()
+        setJourneyTogetherButton()
+    }
+}
+
+extension SessionStartViewController {
+    private func setCollectionView() {
+        view.addSubview(groupCollectionView)
+        groupCollectionView.backgroundColor = .white
+        NSLayoutConstraint.activate([
+            groupCollectionView.topAnchor.constraint(equalTo: view.topAnchor, constant: 20),
+            groupCollectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0),
+            groupCollectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0),
+            groupCollectionView.heightAnchor.constraint(equalTo: groupCollectionView.widthAnchor, multiplier: 0.5)
+        ])
+        groupCollectionView.delegate = self
+        groupCollectionView.dataSource = self
+    }
+    private func setJourneyTogetherButton() {
+        view.addSubview(journeyTogetherButton)
+        NSLayoutConstraint.activate([
+            journeyTogetherButton.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: -120),
+            journeyTogetherButton.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            journeyTogetherButton.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            journeyTogetherButton.heightAnchor.constraint(equalToConstant: 120)  // 버튼의 높이 조절
+        ])
+        journeyTogetherButton.addTarget(self, action: #selector(presentModalQueue), for: .touchUpInside)
+    }
+    @objc private func presentModalQueue() {
+        let modalQueueViewController = ModalQueueViewController()
+        self.present(modalQueueViewController, animated: true)
+    }
+}
+
+extension SessionStartViewController: UICollectionViewDelegateFlowLayout {
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return groupData.count
+    }
+    func collectionView(_ collectionView: UICollectionView,
+                    layout collectionViewLayout: UICollectionViewLayout,
+                    sizeForItemAt indexPath: IndexPath) -> CGSize {
+        return CGSize(width: 80, height: 106)    // 변경 필요
+    }
+}
+
+extension SessionStartViewController: UICollectionViewDataSource {
+    func collectionView(_ collectionView: UICollectionView,
+                        cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = groupCollectionView.dequeueReusableCell(withReuseIdentifier: "groupCell", for: indexPath)
+                    as? GroupCollectionViewCell
```

Comment:
120자 제한으로 인해 개행해주신 것 같은데 조금 더 가독성을 신경써주시면 좋을 것 같아요 !

```suggestion
        let cell = groupCollectionView.dequeueReusableCell(
            withReuseIdentifier: "groupCell",
            for: indexPath
        ) as? GroupCollectionViewCell
```

Reply:
넵 어떻게 개행해줘야 될 지 몰라서 이렇게했었습니다 ㅋㅋㅋ!! 참고하겠습니다!